app/assets/stylesheets/application.scss

```diff
@@ -1,14 +1,128 @@
 // SASS file
 @import 'bootstrap';
 
+//@at-root
+
+$theme-colors: (
+  "dark-purp": #00227B,
+  "medium-purp": #3949AB,
+  "light-purp": #6F74DD,
+  "light-orange": #FFB74D,
+  "yellow": #FFE97D,
+  "dark-yellow": #C88719
+
+);
+
+h1{
+    color: #C88719;
+}
+
+h2{
+    color: #C88719;
+}
+
+a.button{
+    display:inline-block;
+    padding:0.35em 1.2em;
+    margin:0 0.3em 0.3em 0;border-radius:10px;
+    box-sizing: border-box;
+    text-decoration:none;
+    font-weight: bold;
+    color:#000000;
+    text-align:center;
+    transition: all 0.2s;
+    background-color:#FFE97D;
+    border:0.1em solid #FFE97D;
+    text-transform: uppercase;
+    margin-top: 5spx;
+    }
+    a.button:hover{
+    color:#C88719;
+    background-color:#FFffff;
+    border: 2px solid #C88719;
+    }
+    @media all and (max-width:30em){
+    a.button1{
+    display:block;
+    margin:0.4em auto;
+    }
+}
+
+
+.navbar {
+    background-color: #00227B;
+    color: white;
+}
+
 .footer {
     position: fixed;
     display: flex;
     left: 0;
     bottom: 0;
     width: 100%;
-    background-color: #494949;
+    background-color: #00227B;
```

Comment:
It'd be good to start referencing sass variables

Reply:
wow thank you mina!! i just looked up what sass variables are and then got in a rabbithole reading about mixins and my mind is blown! So cool, def going to refactor and set up variables for the colors at least